catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -226,25 +319,49 @@ private String getRegistryId(Metacard mcard) {
                 .toString();
     }
 
-    public void init() throws FederationAdminException {
-
-        List<Metacard> registryMetacards = federationAdmin.getRegistryMetacards();
-
+    public void init()
+            throws UnsupportedQueryException, SourceUnavailableException, FederationException {
+
+        List<Metacard> registryMetacards;
+        Filter registryFilter = filterBuilder.attribute(Metacard.TAGS)
+                .is()
+                .like()
+                .text(RegistryConstants.REGISTRY_TAG);
+        Subject systemSubject = Security.getInstance()
+                .getSystemSubject();
+        QueryImpl query = new QueryImpl(registryFilter);
```

Comment:
We talked about this for quite a while and ended up deciding that for the subscription stuff we would set it at 1000 and have a ticket to come back and fix the subscription stuff to have a 1000 page size for each registry query. In this case we are only doing a local query but we still don't want to end up doing multiple queries. I think a constant of 1000 is fine for now.